### ISSUE TITLE
Calibration file and TF in rosbag 

### ISSUE THREAD

**Issue by icameling** [hemi86/hiltislamchallenge#18]
Hello
The following TF frame is from **Basement_1.bag** 
![Basement_1](https://user-images.githubusercontent.com/12575288/143584479-66251b09-7c76-4a8b-ba45-99a88f2474bc.png)
and the following TF frame is from **Campus_2.bag**
![Campus_2](https://user-images.githubusercontent.com/12575288/143584652-0064c0b6-34a7-41b0-89a3-ea5428e72629.png)

The extrinsic parameters in the provided calibration file `https://storage.googleapis.com/hilti_challenge/Calibration/calibration_02.yaml` seem to be the same as in Campus_2.bag.

I am confused about which extrinsic parameters I should use when I evaluate Basement_1.bag.

**Comment by yhabib29** [comment]
True, the TF in `Basement_1.bag` are different from `calibration_02.yaml` file. Did you manage to find which parameters are the good one ?

**Comment by DesWudio** [comment]
@icameling @yhabib29 Hello, I am also working on this dataset, could you share the "calibration_02.yaml" mentioned above, the url is failed to access for now
